semantic-conventions/src/tests/data/markdown/deprecated/expected.md

```diff
@@ -16,7 +16,7 @@
 
 **[1]:** If `net.transport` is not specified, it can be assumed to be `IP.TCP` except if `http.flavor` is `QUIC`, in which case `IP.UDP` is assumed.
 
-`http.flavor` MUST be one of the following or, if none of the listed values apply, a custom value:
+The following values are defined for `http.flavor`. If one of them applies, then it MUST be used, otherwise a custom value MAY be used.
```

Comment:
I think I would still like to have the name of the attribute being described stand out more.

One suggestion is to add a section to https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/common.md or a similar place to explain the concept of "open enums" (name up for debate). Then we could write "`http.flavor` is an open enum (<-- link to definition here) with the following non-exhaustive list of possible values:"


Or we could simply do:

```suggestion
`http.flavor` SHOULD be set to one of the following values:
```

And for allow_custom_values=false, we use MUST instead of should.

If you keep it like it is now with the attribute name not at the start, I suggest to at least make the attribute name bold by adding `**` around it.

Reply:
ptal, it now reads:

`http.flavor` has the following list of well-known values. If one of them applies, then it MUST be used, otherwise a custom value MAY be used.